File: 5KYU/Perimeter of squares in a rectangle/solution.cpp
typedef unsigned long long ull;
class SumFct
{
  public:
  static ull perimeter(int n)
  {
    ull total = 0;
    ull a = 4;
    ull b = 4;
    
    for(auto i = 0; i <= n; i++ ) {
       total += a;
       auto c = a + b;
       a = b;
       b = c;
    }
    return total;
  }
};

File: 5KYU/Perimeter of squares in a rectangle/solution.js
function perimeter(n) {
  
    let total = 0;
    let a = 4;
    let b = 4;
    
    for(i = 0; i <= n; i++) {
      total += a;
      let c = a + b;
      a = b;
      b = c;
    }
    return total;
}